doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-disabling-security-management-proc.adoc

```diff
@@ -0,0 +1,10 @@
+[id='business-central-disabling-security-management-proc']
```

Comment:
@RishiRajAnand Same question as for the `Enabling the security management feature`. Do we need this chapter? Is there any use case where a user wants to create own security implementation?

Reply:
@kaldesai Could you please remove this file as it is not used anymore?

Reply:
@tomasdavidorg, I have removed this file.